### PR TITLE
Who we are content restructure

### PR DESCRIPTION
**Pull request by yndajas**
Dawn's work on restructuring the "Who we are" section, including renaming it to "About us"

### CODE REVIEW

File: src/about-us/how-dxw-is-run.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+---
```
Comment (snim2) [inline src/about-us/how-dxw-is-run.md]: There are three empty files added in the first commit, was that intended?

Comment (yndajas) [reply src/about-us/how-dxw-is-run.md]: I think Dawn was creating the structure in one commit and the content in another. We could rebase this, but given content changes often come in a series of non-atomic commits from the CMS, I'm tempted not to bother in this context. Happy to squash some of the commits though if preferable

File: src/about-us/who-we-are.md [hunk 2]
```diff
@@ -11,4 +11,22 @@ Our approach is to add value early and often, rapidly researching, testing, lear
...
+
+Sales: generates a pipeline of work that’s on mission and provides work for the people we have.
+
+Delivery+: specialises in creating bespoke digital transactional services by providing multidisciplinary teams
+
+GovPress: designs, builds, hosts and supports content-first websites for public and third sector clients
+
+Strategy: helps organisations form strategies, plans and business cases, and build the capability needed for digital transformation
+
+Product and Performance
```
Comment (snim2) [inline src/about-us/who-we-are.md]: Have we missed some text here?

Comment (yndajas) [reply src/about-us/who-we-are.md]: (I am tempted to do some squashing here but I really don't have the dxw time!)

**Comment by yndajas** [comment]
> This is not easy to review, but I think it's good to merge!

As discussed, have squashed Dawn's eight content commits and my four tidy up commits into a single summarised commit to keep the history more straightforward for future reference. Thanks for the review!

**Pull request merged**